.github/workflows/e2e-android.yml

```diff
@@ -17,6 +22,25 @@ jobs:
     # If it takes longer it usually fails, so no need to pay for more
     timeout-minutes: 25
     steps:
+      - uses: google-github-actions/auth@v2
+        if: ${{ env.SECRETS_AVAILABLE }}
+        with:
+          project_id: celo-mobile-mainnet
+          credentials_json: ${{ secrets.MAINNET_SERVICE_ACCOUNT_KEY }}
+      - name: Google Secrets
+        if: ${{ env.SECRETS_AVAILABLE }}
+        id: google-secrets
+        uses: google-github-actions/get-secretmanager-secrets@v2.1.4
+        with:
+          secrets: |-
```

Comment:
I was looking to minimize the work required to run e2e tests locally for developers. I'll remove this and provide an example .env with setup instructions.

With Mobilestack, we may want to go back to using .env and repository-level secrets so partners can manage and fund their own e2e test accounts instead of sharing e2e wallet credentials, but that's a separate piece of work.

Reply:
Changed in dfc681efe80290e428ca2ea718939ddc3a822a96.